_locales/en/messages.json

```diff
@@ -10,6 +10,7 @@
   "config.ieargs.label.before": { "message": "Command Line Arguments:" },
   "config.ieargs.label.after":  { "message": "" },
   "config.contextMenu.label":   { "message": "Add \"Open by IE\" items to the context menu" },
+  "config.mainFrame.label":     { "message": "Only check URL which is shown in location bar" },
```

Comment:
`onlyMainFrame` seems more meaningful instead of `mainFrame`.

Reply:
Thanks, I've fixed it.